video/agon_fonts.h

```diff
@@ -11,6 +11,7 @@
 // 05/09/2022:		Renamed file
 // 20/02/2023:		Marked out non-standard CP-1252 characters in comments, fixed £ and `
 // 14/10/2023:		LB: Added CP1252 characters from BBC Basic for SDL2, kept all Agon ASCII characters as is, including £
+// 29/07/2025:		SS: Added "Graphics Extended" CP1252 characters with some adjustments as outlined

```

Comment:
as we share the same initials and my name is in the "author" comment above you _might_ want to change this comment, and maybe add your name into the authors comment 😀